pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -214,6 +233,27 @@ func (a ClusterOperatorAnonymizer) Marshal(_ context.Context) ([]byte, error) {
 	return runtime.Encode(serializer, a.ClusterOperator)
 }
 
+func isHealthyOperator(operator *configv1.ClusterOperator) bool {
+	for _, condition := range operator.Status.Conditions {
+		switch {
+		case condition.Type == configv1.OperatorDegraded && condition.Status == configv1.ConditionTrue,
+			condition.Type == configv1.OperatorAvailable && condition.Status == configv1.ConditionFalse:
```

Comment:
Upgradeable == false seems like a reason to collect

Reply:
or do we think that the additionally collected data would not be useful in any way to resolve such a situation?

Reply:
I *think* we don't need to look for failing pods in this case.  Fortunately would be easy to fix if we do.